src/main/java/io/cdap/plugin/gcp/bigquery/sink/AbstractBigQuerySinkConfig.java

```diff
@@ -66,6 +66,7 @@
   protected String gcsChunkSize;
 
   @Macro
+  @Nullable
   @Description("Whether to modify the BigQuery table schema if it differs from the input schema.")
   protected boolean allowSchemaRelaxation;
```

Comment:
does this work even if it's a boolean and not a Boolean? Will it get defaulted to false if no value is given?

Reply:
Modified it to be `Boolean` not sure why we used `boolean` before.